CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ option(INFO "Enable gathering of math library information" OFF)
 option(ENABLE_COVERAGE "Enable coverage build." OFF)
 option(ENABLE_LIBRI "Enable EXX with LibRI." OFF)
 option(USE_ELPA "Enable ELPA" ON)
+option(USE_ABACUS_LIBM " Enable abacus self libmath." ON)
```

Comment:
Since it pass the test with GCC/clang, and it’s speed is always better(faster than the latest libm),  I recommend to enable it by default and no need to detect the library version.
On the other hand, not all compilers will link with libm.